### ISSUE TITLE
Make doesn't work

### ISSUE THREAD

**Issue by Scotts-Thoughts** [DanielOlivaw/pokecrystal16#1]
I receive an error with an unmodified version of the repo.

```
rgbgfx  -o gfx/mobile/up_arrow.2bpp gfx/mobile/up_arrow.png
FATAL: Image height (4 pixels) is not a multiple of 8!
Conversion aborted after 1 error
make: *** [Makefile:184: gfx/mobile/up_arrow.2bpp] Error 1
```

Btw your work here is incredible, I'd love to chat about it and ask some questions!

**Comment by DanielOlivaw** [comment]
That's a weird error I don't think I've run into before. I don't think I've ever made edits to that part of the code so I don't know why it would be giving you issues. All I can suggest is to try replacing gfx/mobile/up_arrow.png with a version that includes an extra 4 pixels of whitespace and see if that lets you bypass the problem.

**Comment by Scotts-Thoughts** [comment]
Yeah, I'm getting the same error with the base repo. I wonder why it isn't working. I can easily compile the newer disassembly.

**Comment by Scotts-Thoughts** [comment]
Replacing the graphics with the ones from the most up-to-date pokecrystal disassembly fixed the issue.

**Comment by 0bocker** [comment]
@Scotts-Thoughts but then dont you lose the gfx for all the new mons?

**Comment by FrigidLava** [comment]
I get that it doesn't have -L as an option when I try to make it.

**Comment by fearjunkie** [comment]
> I get that it doesn't have -L as an option when I try to make it.

Yeah, I'm getting this error too.

**Comment by SharathHebbar** [comment]
try rgbasm 0.5.2 and then compile
